book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  attr_accessor :title, :author, :rentals
```

Comment:
[OPTIONAL]

- Same here as well. Kindly add the getters and setters at the top of the def initialize method below the class.